notebooks/examples/tabular_data_processing.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Intro\n",
+    "\n",
+    "This notebook shows an example how to use HexTractor to transform tabular data to heterogeneous graph."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load libs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import rootutils\n",
+    "import autoroot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import hextractor.data_sources as data_sources\n",
+    "import hextractor.structures as structures\n",
```

Comment:
why not use normal import instead of import aliases
```diff
-import hextractor.data_sources as data_sources
-import hextractor.structures as structures
+from hextractor import data_sources
+from hextractor import structures

Reply:
Yeah right. I prefer to import full modules so you can easily reload them in the notebook if you change the implementation. So if you do:
```
import module as m

m.foo() # initial results

############################
# change the implementation
# in the module "m" later
############################

from importlib import reload

m = reload(m)
m.foo() # different results
```

That's the reason, why I prefer to use `import module as m` in notebooks :)